package.json

```diff
@@ -26,7 +26,8 @@
         "json5": "^2.1.3",
         "notp": "^2.0.3",
         "phaser": "3.54.0",
-        "webfontloader": "^1.6.28"
+        "webfontloader": "^1.6.28",
+        "games-automation-stathelper": "git+ssh://git@github.com/bbc/games-automation-stathelper.git#main"
```

Comment:
this will need to be `"games-stat-helper": "github:bbc/games-automation-stathelper.git"` to work with newer NPM

Reply:
also should it be a devDep?